public/index.htm

```diff
@@ -83,7 +83,7 @@
       </div>
     </div>
 
-    <div id="newWidget" title="Configure a New Widget">
```

Comment:
I removed this, because it will be better defined in the JavaScript. The same dialog is being used for adding widgets as well as configuring them, so the title changes every time it's opened. I'll explain more of that later